modules/apps/object/object-test/src/testFunctional/paths/ObjectNotifications.path

```diff
@@ -0,0 +1,46 @@
+<html>
+<head>
+<title>ObjectNotifications</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">ObjectNotifications</td></tr>
+</thead>
+
+<tbody>
+	<tr>
+		<td>DATA_SOURCE_ATTACHMENTS</td>
+		<td>//div[@tabindex='0']//span[contains(.,'${key_dataSource}')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>DATA_SOURCE_ATTACHMENTS_SELECT_OBJECT</td>
+		<td>//div[@class='dropdown-menu show']//div[@class='d-flex justify-content-between' and contains(.,'Custom Object 161924')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>FIELD_ATTACHMENTS_OBJECT</td>
+		<td>//div[@class='input-group-item']//div[@class="form-control form-control-tag-group input-group"]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>FIELD_ATTACHMENTS_OBJECT_SELECT_FIELD</td>
+		<td>//span[@class='custom-control-label']//span[@class="custom-control-label-text"]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>TEMPLATE_NAME</td>
+		<td>//div[@class='table-list-title']//a[contains(.,'${key_templateName}')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>DELETE_ATTACHMENT_BUTTON</td>
+		<td>//*[@class='lexicon-icon lexicon-icon-times-small']</td>
+		<td></td>
+	</tr>
```

Comment:
I feel that we should have existing paths that work. Can you double check this?

cc: @evanilson-santana

Reply:
I'll verify this